README.md

```diff
@@ -102,13 +107,30 @@ use the const `COMMON_CONTEXT` to make some variables available for all template
     send_notification("foo@bar.com", "event_created", context, [attachment])
     
     ```
-   
+
 8. By default, notifications will be sent immediately, if you only want to add notification to the message queue
  and send it later, configure `ILMOITIN_QUEUE_NOTIFICATIONS`:
     ```python
     ILMOITIN_QUEUE_NOTIFICATIONS = True
     ```
 
+## Using the GrapQL API
+1. Add the `Query` to the entrypoint where you build your schema:
+
+```python
+# my_app/schema.py
+import django_ilmoitin.api.schema as django_ilmoitin_schema
+
+class Query(
+    # other extended classes
+    django_ilmoitin_schema.Query,
+    graphene.ObjectType,
+):
+    pass
+
+```
```

Comment:
If one wants the API enabled but is not familiar with Graphene, this will probably look pretty cryptic. Also, she cannot just copy-paste the example to get a working API, but will need all kinds of other Graphene specific stuff to be set up too. So at minimum, I would add a remark about Ilmoitin using Graphene for the API, and a link to Graphene setup docs.

Ideally, I think it should be possible to also enable the API easily without knowing or caring about Graphene at all, with just something like 

``` python
# project/urls.py
...
urlpatterns += path('ilmoitin/', include('ilmoitin.urls'))
```
Dunno if implementing that is actually worth the trouble though, probably not at least right now 🙂

Reply:
Fair point. I think the easiest option would be to add some more details on the docs about the API requiring a working `graphene` API and link to their docs.

The logic we were following for this was to have a separate "plug-n-play" schema which you could mount in an existing GraphQL (graphene) API, so you wouldn't need to have a different endpoint _just_ for `ilmoitin`.

This implementation doesn't have everything it requires to have a standalone Graphene server, so it would require more work to have the `path('ilmoitin/', include('ilmoitin.urls'))` which would have the API out of the box.

cc @igordavydsson

Reply:
yeah, very good idea to have the fully functional API within ilmoitin, that you could just connect to a url path within your project 👍  it would indeed require a bit more work though, so maybe we could put it on hold for now. I think the docs that Alex added point the potential user to the right direction of installing graphene and setting up a working API first, so that's probably enough for now. 🙂